packages/edit-post/README.md

```diff
@@ -507,6 +507,18 @@ _Parameters_
 -   _settings_ `?Object`: Editor settings object.
 -   _initialEdits_ `Object`: Programmatic edits to apply initially, to be considered as non-user-initiated (bypass for unsaved changes prompt).
 
+<a name="store" href="#store">#</a> **store**
```

Comment:
Let's don't expose this store for now and leave for later to decide if we want to do it.

Reply:
Is it ok to remove the `export` statement from the package? 
Considering that after removing the reference from both `editor` and `block-library` packages, there will still be references to it in the `block-directory` package... Will it still be accessible if I remove it?

Reply:
Yes, let's revert that as well. It's a bit unfortunate but let's leave as the very last task of the refactoring process.

Reply:
You mean revert the `block-directory` commit as well or only the `editor` + `block-library` ones? 
I asked about the `export` statement because I thought that by removing it, some other bugs could be caused.

Reply:
I missed your last comment. I'm sorry about it. I applied all changes myself:

I only reverted changes to `@wordpress/editor` and `@wordpress/block-library` packages. I contemplated on using `export` for `store` on `@wordpress/edit-post` and I think you were right, it makes sense for 3rd party plugin. `@wordpress/block-directory` is an internal plugin so it makes sense to keep it working this way.